core/src/Data/Query/IdsQuery.php

```diff
@@ -1,10 +1,10 @@
 <?php
 
-namespace Harmony\Core\Repository\Query;
+namespace Harmony\Core\Data\Query;
 
 class IdsQuery implements Query {
```

Comment:
should inherit KeysQuery. But what is the goal of KeyQuery?

Reply:
Fixed.